evm/its.js

```diff
@@ -419,15 +430,32 @@ async function processCommand(_axelar, chain, chains, action, options) {
                 return;
             }
 
-            const data = [];
+            if (itsVersion === '2.1.1') {
+                const trustedAddresses = [];
+                for (const trustedChain of trustedChains) {
+                    const chainConfig = chains[trustedChain];
+                    if (chainConfig?.contracts?.InterchainTokenService?.address) {
+                        trustedAddresses.push(chainConfig.contracts.InterchainTokenService.address);
+                    } else {
+                        throw new Error(`No InterchainTokenService address found for chain ${trustedChain}`);
+                    }
+                }
+                
+                for (const [trustedChain, trustedAddress] of trustedChains.map((chain, index) => [chain, trustedAddresses[index]])) {
+                    const tx = await interchainTokenService.setTrustedAddress(trustedChain, trustedAddress, gasOptions);
```

Comment:
```suggestion
                    const tx = await interchainTokenService.setTrustedAddress(trustedChain, 'hub', gasOptions);
```
trusted address should be set to `hub`, we're only adding amplifier chains now, so we can assume that to remove all this unnecessary complexity